caf-audit/src/main/java/com/hpe/caf/auditing/AuditEventBuilder.java

```diff
@@ -38,6 +38,13 @@ void addEventParameter(
         String value
     );
 
+    void addEventParameter(
+            String name,
+            String columnName,
+            String value,
+            AuditIndexingHint indexingHint
+    ) throws Exception;
```

Comment:
@dermot-hardy Sorry to ask but can you explain the last point please about the default implementation? I'm not sure what you mean exactly.